File: oner-php/app/Employee.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Employee extends Model 
{    
    protected $table = 'employee';

    const CREATED_AT = 'created_time';
    const UPDATED_AT = 'updated_time';
    
    
    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = [
        'employee_identification_number',
        'name',
        'gender',
        'education',
        'join_date',
        'permanent_date',
        'marital_status',
        'number_of_children',
        'live_with_parent',
        'live_with_spouse_parent',
        'phone_number',
        'emergency_contact_phone',
        'date_of_birth',
        'nrc_number',
        'bank_account_number',
        'tax_id',
        'passport_number',
        'address',
        'profile_image',
        'position_id',
        'department_id',
        'status',
        'created_user_login_id',
        'updated_user_login_id',
    ];

    protected $hidden = [
        'department_id',
        // 'employee_id'
    ];
    public function department(){
        return $this->hasOne("App\Department",'id','department_id');
    }
    public function position(){
        return $this->hasOne("App\Position",'id','position_id');
    }
}

File: oner-python/src/app/infrastructure/models/HR/employee.py
from sqlalchemy import Column, String, Enum, Boolean, Integer,Date
from sqlalchemy.orm import relationship
from sqlalchemy.sql.schema import ForeignKey
from app.core.Entity.HR.employee import MaritalStatus, Status
from app.infrastructure.models.baseModel import BaseModel

class Employee(BaseModel):
    __tablename__='employee'

    name = Column(String)
    gender = Column(String)
    education = Column(String)
    marital_status = Column(Enum(MaritalStatus))
    number_of_children = Column(Integer)
    live_with_spouse_parent = Column(Boolean)
    phone_number = Column(String)
    emergency_contact_phone = Column(String)
    date_of_birth = Column(Date)
    nrc_number = Column(String)
    bank_account_number = Column(String)
    address = Column(String)
    position_id = Column(Integer, ForeignKey('position.id'))
    department_id = Column(Integer, ForeignKey('department.id'))
    status = Column(Enum(Status))

    position = relationship('Position', uselist=False)
    department = relationship('Department', uselist=False)